src/main/java/appeng/items/storage/BasicItemStorageCell.java

```diff
@@ -33,12 +33,12 @@
 public final class BasicItemStorageCell extends AbstractStorageCell<IAEItemStack>
 {
 
-
 	protected final int perType;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this "protected" modifier. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2156)